### PR TITLE
Upgrade framework and nuget

### PR DESCRIPTION
**Pull request by dmitry-livchak**

### CODE REVIEW

File: AWSDeleteStack/AWSDeleteStack.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>AWSDeleteStack</id>
+    <version>1.4.2</version>
+    <title>AWSDeleteStack</title>
+    <authors>TravCorp</authors>
+    <owners>TravCorp</owners>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Wrapper around AWS CloudFormation API. Deletes a stack and outputs the restlt</description>
```
Comment (bcipri) [inline AWSDeleteStack/AWSDeleteStack.nuspec]: There is a typo here: "restlt"

**Comment by joshski** [comment]
Seems reasonable to me. We are downgrading the AWS stuff from the beta release, right. Any particular reason for that?

**Comment by dmitry-livchak** [comment]
Yes, If I honestly add beta into Nuget dependencies, Nuget is demanding to mark my package as pre-release as well. It was previously pointing to a deprecated AWS v2 package.

File: AWSDeleteStack/AWSDeleteStack.nuspec (hunk 1 above)
Comment (dmitry-livchak) [inline AWSDeleteStack/AWSDeleteStack.nuspec]: yes

**Pull request merged**